### ISSUE TITLE
About dot product(teacher's feedback)

### ISSUE THREAD

**Issue by dongkwan-kim01** [kekmodel/MPL-pytorch#16]
<img width="1017" alt="image" src="https://user-images.githubusercontent.com/49050419/117596378-33eb8200-b17e-11eb-9687-5e046934b34d.png">

From the above equation, the dot product computes with
1. labeled data and target in time t+1
2. unlabeled data and pseudo label in time t

In the main.py function,
s_loss_l_old = F.cross_entropy(s_logits_l.detach(), targets) 
Should it be changed to CE(s_logits_us, pseudo_label) ?
